cmd/timoni/bundle_apply.go

```diff
@@ -222,11 +169,16 @@ func applyBundleInstance(name, namespace, moduleURL, moduleVersion string, value
 		return err
 	}
 
+	if instance.Module.Digest != "" && mod.Digest != instance.Module.Digest {
+		return fmt.Errorf("the upstream digest of version %s doesn't match the specified digest %s",
```

Comment:
Maybe mention the digest here of upstream which causes the mismatch?

Reply:
Done! Thanks